doc/rm/c_cpp_coding_style.md

```diff
@@ -110,6 +110,20 @@ This convention helps readers distinguish which files they should not expect to
 
 The above rules also do not apply to system includes, which should be included by the names dictated by the ISO standard, e.g. `#include <stddef.h>`.
 
+### Linker Script Provided Definitions
+
+Some C/C++ programs may need to use symbols that are defined by a linker script.
```

Comment:
```suggestion
Some C/C++ programs may need to use symbols that are defined by a linker script or in an external assembly file.
```

Reply:
If we're going to mention this, I would point out that these are symbols defined by `.set` and `.int`, not labels.

Reply:
Do you really mean `.int`?

Reply:
No, apparently not. I think I'm thinking of some non-gas assembler syntax. But definitely `.set + .global`.